### PR TITLE
Channel Info Screen UI and all implementations

### PR DESCRIPTION
**Pull request by developerBarak**
channel info screen

### CODE REVIEW

File: lib/ui/nav_pages/you_page/you_page_viewmodel.dart [hunk 1]
```diff
@@ -23,6 +23,8 @@ class YouPageViewModel extends BaseViewModel {
   final _snackBar = locator<SnackbarService>();
   final _connectivityService = locator<ConnectivityService>();
   final _apiService = ZuriApi(coreBaseUrl);
+  // final _storageServe = locator<StorageService>
```
Comment (pauleke65) [inline lib/ui/nav_pages/you_page/you_page_viewmodel.dart:26]: Please remove

File: lib/ui/view/channel/channel_info/widgets/sixth_section.dart [hunk 2]
```diff
@@ -15,45 +15,47 @@ class SixthSection extends ViewModelWidget<ChannelInfoViewModel> {
...
+                    Text(
+                      'Archive Channel',
+                      style:
+                          AppTextStyles.archiveTextStyle2.copyWith(fontSize: 14),
+                    ),
+                    const SizedBox(height: 6,),
+                    Container(
+                        alignment: Alignment.center,
+                        child: Text(
+                          'Archiving the channel will remover it from the channel list, and close it from all members.'
```
Comment (pauleke65) [inline lib/ui/view/channel/channel_info/widgets/sixth_section.dart:50]: Move to app_strings.dart

File: test/view_models/channel_info_view_model_test.dart [hunk 3]
```diff
@@ -23,7 +23,7 @@ void main() {
           'When setChannelName is called and the channel name is set, should be true.',
           () async {
         final model = ChannelInfoViewModel();
-        model.setChannelName('team-zuri-app');
+        // model.setChannelName('team-zuri-app');
```
Comment (pauleke65) [inline test/view_models/channel_info_view_model_test.dart:26]: Remove comment

**Review by pauleke65 — CHANGES_REQUESTED** [review]
Minor changes requested

**Review by jeremiahseun — APPROVED** [review]
LGTM

**Review by YenumJay — APPROVED** [review]

**Pull request merged**